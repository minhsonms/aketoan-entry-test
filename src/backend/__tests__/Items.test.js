/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Items from '../test/Items';

// Snapshot: test that renders the Items component with no items passed
it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Items />).toJSON();
  expect(tree).toMatchSnapshot();
});
