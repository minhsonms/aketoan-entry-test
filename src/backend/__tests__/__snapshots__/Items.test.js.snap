// Jest Snapshot v1, https://goo.gl/fbAQLP


exports[`renders correctly when there are no items 1`] = `
<span
  className="empty-message"
>
  No items in list
</span>
`;
